### ISSUE TITLE
the info need me to tell you

### ISSUE THREAD

**Issue by erquren** [tensorflow/tensorflow#41890]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- OS Platform and Distribution win10 18363:
- TensorFlow installed from conda:
- TensorFlow version 2.3.0-rc0:
- Python 3.7:
- CUDA/cuDNN 10.2:
- 1080  16G:

**Describe the current behavior**
Epoch 1/5
WARNING:tensorflow:AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x0000013933B3A798> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
WARNING: AutoGraph could not transform <function Model.make_train_function.<locals>.train_function at 0x0000013933B3A798> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: 'arguments' object has no attribute 'posonlyargs'
To silence this warning, decorate the function with @tf.autograph.experimental.do_not_convert
235/235 [==============================] - 0s 1ms/step - loss: 0.7901 - accuracy: 0.7434
Epoch 2/5
235/235 [==============================] - 0s 1ms/step - loss: 0.5730 - accuracy: 0.8110
Epoch 3/5
235/235 [==============================] - 0s 1ms/step - loss: 0.5279 - accuracy: 0.8239
Epoch 4/5
235/235 [==============================] - 0s 1ms/step - loss: 0.5032 - accuracy: 0.8309
Epoch 5/5
235/235 [==============================] - 0s 1ms/step - loss: 0.4875 - accuracy: 0.8352
**Describe the expected behavior**

**Standalone code to reproduce the issue**
import tensorflow as tf
from tensorflow import keras
fashion_mnist = keras.datasets.fashion_mnist
(x_train, y_train), (x_test, y_test) = fashion_mnist.load_data()
x_train = x_train / 255.0
x_test = x_test / 255.0

model = keras.Sequential([
    keras.layers.Flatten(input_shape=(28, 28)),
    keras.layers.Dense(10, activation=tf.nn.softmax)
])
model.compile(optimizer=tf.keras.optimizers.SGD(0.1),
              loss = 'sparse_categorical_crossentropy',
              metrics=['accuracy'])
model.fit(x_train,y_train,epochs=5,batch_size=256)

and now it`s happen

**Comment by Saduf2019** [comment]
@erquren 
I ran the shared code and do not face any warnings,please find [gist here for 2.2](https://colab.research.google.com/gist/Saduf2019/89f7c21c1ee3b3fcb6f45e2c1c7b3cf8/untitled305.ipynb) and [2.3 as reported](https://colab.research.google.com/gist/Saduf2019/580d82953532a58f3e8db7cee73c2315/untitled305.ipynb).

You can safely ignore the warning log as its intended to debug logging in AutoGraph issues. Thanks !

You may refer to resolved issues with same error:
#37144 [link](https://github.com/tensorflow/tensorflow/issues/38947#issuecomment-620707630) [link1](https://github.com/tensorflow/tensorflow/issues/41120#issuecomment-654268902) [link2](https://github.com/tensorflow/tensorflow/issues/32377#issuecomment-529893477)

**Comment by mdanatg** [comment]
This looks like a gast version mismatch. If the issue still reproduces in tf-nightly, please re-open the bug. Thanks!
